scipy/sparse/linalg/isolve/_gcrotmk.py

```diff
@@ -270,10 +270,12 @@ def gcrotmk(A, b, x0=None, tol=1e-5, maxiter=1000, M=None, callback=None,
         raise ValueError("Invalid value for 'truncate': %r" % (truncate,))
 
     if atol is None:
+        '''
```

Comment:
This change looks unintentional.

Reply:
Thank you for your kind suggestions, the comment was added in debugging, and has been deleted. I'll submit it later.